examples/guides/messaging/messaging-with-kafka/README.md

```diff
@@ -100,6 +100,45 @@ The `topics` field specifies the topics that must be listened by this consumer.
 
 The `pollingInterval` field is the time interval that a consumer polls the topic. 
 
+Please find other subscriber configuration parameters below:
+
+| Parameter | Description  |
+| :---   | :- |
+| offsetReset | Offset reset strategy if no initial offset. |
```

Comment:
Should we give all these configurations in the guide or in the module readme?

Reply:
Added in the Module.md as well.
https://github.com/wso2-ballerina/module-kafka/pull/45

Reply:
Shall we have the detailed parameter list in the module.md? Since this is a guide we don't need to put all the values here